install.test.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
I would really like to avoid writing more shell scripts than necessary.
We can merge this, but please open an issue to rewrite this in an actual language.